Examples/Python/python/acts/examples/simulation.py

```diff
@@ -532,6 +533,7 @@ def addSimWriters(
     s: acts.examples.Sequencer,
     simHits: str = "simhits",
     particlesSimulated: str = "particles_simulated",
+    field: acts.MagneticFieldProvider = None,
```

Comment:
should `writeHelixParameters` also be wired here? otherwise it cannot be used right now I think